### ISSUE TITLE
Please consider moving the apt-key to an SSL-controlled host

### ISSUE THREAD

**Issue by oskarpearson** [sensu/sensu-docs#159]
Hi

Please could you consider moving your documentation and apt key to an SSL repository?

http://sensuapp.org/docs/0.16/packages includes this command line:
```shell
wget -q http://repos.sensuapp.org/apt/pubkey.gpg -O- | sudo apt-key add -
```
Since this doesn't use SSL, it's possible to man-in-the-middle the repo and gain root access on the servers running Sensu.

Thanks!

Oskar

**Comment by calebhailey** [comment]
Whoa, we still haven't fixed this. We should be able to make these changes in the impending 0.23 docs.

**Comment by calebhailey** [comment]
This has been overlooked for a loooooong time, but will be fixed by #408.
